code/game/machinery/doors/brigdoors.dm

```diff
@@ -101,6 +119,8 @@
 	if(stat & (NOPOWER|BROKEN))
 		return 0
 
+	print_report()

```

Comment:
Could you perhaps make this required to actually start the timer and close the cell door? You could make the print_report proc return 1 if they type something into both prompts (check the variables for values), otherwise return 0, and then convert this line into something like
```
if(!print_report())
   return
```
So it doesn't actually close the doors if they didn't enter stuff.